docs/review-process.md

```diff
@@ -79,6 +79,11 @@ the end-user getting your improvement faster.
    Be sure to add a motivation (or use case) to your PR, so the reviewer can
    understand why you are making this change (or why you make certain decisions).
 
+7. **Comply with Architecture Decisions.**
```

Comment:
This should be the first thing in the thought process after the "problem" that the PR should solve is identified. I'd move it up to the top.

Reply:
I moved it to a new paragraph before coming to the PR section.